### ISSUE TITLE
"Run Single File" does not support CLI-Arguments with spaces

### ISSUE THREAD

**Issue by nigjo** [apache/netbeans#4622]
### Apache NetBeans version

Apache NetBeans 14

### What happened

In the Properties window of a "runnable" java source file there is a setting for "Arguments". These arguments are handed over to the file when run as "single file". But this property does not handle spaces in arguments properly.

### How to reproduce

    Arguments=first second third

works good. in `main` the `args` array has 3 entries as expected

    Arguments="A first argument" second "All good are three"

does not work as expected. `args` will have 8 entries.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

windows 10

### JDK

jdk 11

### Apache NetBeans packaging

Apache NetBeans binary zip

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by lkishalmi** [comment]
What build system are you using?

**Comment by nigjo** [comment]
There is no project involved. It's the Java 11+ feature to run a .java file from command line.

**Comment by mbien** [comment]
```java
public class NewJavaFile {
    public static void main(String args[]) {
        System.out.println("arguments:");
        for (String arg : args) {
            System.out.println("arg: "+arg);
        }
    }
}
```
```
expected:
$ java NewJavaFile.java "one two" three
arguments:
arg: one two
arg: three
```

in NB:
![image](https://github.com/apache/netbeans/assets/114367/63249e66-c3f6-44fd-9738-a4bb1d508aff)
```
arguments:
arg: "one
arg: two"
arg: three
```

**Comment by mbien** [comment]
I can't reproduce this anymore with NB 20.

the output for `"one two" three` is now:
```
arguments:
arg: one two
arg: three
```

Can someone confirm? 

I am curious how this got fixed, might be the javac update or something else :)
the attribute storage did not change between 19 and 20:
```xml
<attr name="single_file_run_arguments" stringvalue="&quot;one two&quot; three"/>
```

**Comment by mbien** [comment]
fixed by #6262, closing
